src/theme/components/styledList.ts

```diff
@@ -0,0 +1,26 @@
+import { baseListStyles, baseUnorderedStyles } from "./list";
+
+const StyledList = {
+  baseStyle: ({ textSize = "default" }) => ({
+    fontSize: `text.${textSize}`,
+    ...baseListStyles(false, true),
+    ...baseUnorderedStyles(true),
+  }),
+  variants: {
+    capped: {
+      li: {
+        borderLeft: "1px solid",
+        borderRight: {
+          base: "1px solid",
+          md: "none",
+        },
+        borderColor: "ui.border.default",
+        my: "xs",
+        px: "xs",
+        width: "fit-content",
```

Comment:
Sorry, I missed this earlier.  This can be removed altogether.  The mobile UI should span the full width of the window and for desktop it does not impact anything.

Reply:
That's supposed to be an embarrassed emoji, but I don't think it's all that convincing.